.buildkite/pipeline.yml

```diff
@@ -843,6 +843,12 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: ":fire: Flame graph: perf target (diagnostic edmfx)"
+        command: "julia --color=yes --project=perf perf/flame.jl --job_id flame_perf_target_diagnostic_edmfx --target_job diagnostic_edmfx_aquaplanet"
+        artifact_paths: "flame_perf_target_diagnostic_edmfx/*"
+        agents:
+          slurm_mem: 100GB
```

Comment:
I don't think this is a good idea

Reply:
I agree, it was the only way I could get the job to complete. Maybe we should hold off on adding this perf test until I figure out what's going on with the ifelse allocations?